ext/api_util.cpp

```diff
@@ -67,5 +67,7 @@ void export_api_util()
         .def("get_user_connect_timeout", &Tango::ApiUtil::get_user_connect_timeout)
 
         .def("get_ip_from_if", &Tango::ApiUtil::get_ip_from_if)
+        .def("cleanup", &Tango::ApiUtil::cleanup)
+        .staticmethod("cleanup")
```

Comment:
Would you mind adding a short explanation about how this should be used in the howto?  You have most of the code in issue #259 already.  I suggest adding a new section like "Using clients with multiprocessing".  A good spot is just before the [original API section](https://github.com/tango-controls/pytango/blob/bdb60ae/doc/howto.rst#write-a-server-original-api) (since the rest of the doc is more for historical purposes)